proto/pdpb.proto

```diff
@@ -104,6 +104,11 @@ service PD {
     rpc SetExternalTimestamp(SetExternalTimestampRequest) returns (SetExternalTimestampResponse) {}
 
     rpc GetExternalTimestamp(GetExternalTimestampRequest) returns (GetExternalTimestampResponse) {}
+
+    // Get the minimum timestamp across all keyspace groups from API server
+    // TODO: Currently, we need to ask API server to get the minimum timestamp.
+    // Once we support service discovery, we can remove it.
+    rpc GetMinTimestamp (GetMinTimestampRequest) returns (GetMinTimestampResponse) {}
```

Comment:
Shall we call it GetMinTS (following the name conversion of SyncMaxTS())? I have already added GetMinTS to tsopb. The API leader server side calls tsopb.GetMinTS when processing GetMinTimestamp named here. Shall we rename it to GetMinTS() since tsopb.GetMinTS already merged? I raise this concern but leave the decision to you. I'm ok to either way.